gcp/observability/opencensus.go

```diff
@@ -130,11 +140,15 @@ func stopOpenCensus() {
 	if exporter != nil {
 		internal.ClearGlobalDialOptions()
 		internal.ClearGlobalServerOptions()
+		// This Unregister call guarantees the data recorded gets sent to exporter,
+		// synchronising the view package and exporter.
```

Comment:
As with the comment below, I assume this is also safe to do even if the views weren't already registered (i.e. metrics are disabled)?

Reply:
Luckily, yes. Like below, added a docstring.

Reply:
https://github.com/census-instrumentation/opencensus-go/blob/master/stats/view/worker_commands.go#L92 here is the check which makes it safe.